src/app/src/test/kotlin/org/imperial/mrc/hint/unit/controllers/ErrorReportControllerTests.kt

```diff
@@ -47,6 +58,47 @@ class ErrorReportControllerTests
         assertThat(result.statusCode).isEqualTo(HttpStatus.BAD_REQUEST)
     }
 
+    @Test
+    fun `can serialize request body`()
```

Comment:
I think this test would have more value as an integration test i.e. using `testRestTemplate.postForEntity()` rather than `sut.postErrorReport()`